src/server/main_service.cc

```diff
@@ -815,6 +815,7 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
   config_registry.RegisterMutable("replication_timeout");
   config_registry.RegisterMutable("table_growth_margin");
   config_registry.RegisterMutable("tcp_keepalive");
+  config_registry.RegisterMutable("managed_service_info");
```

Comment:
I do not think we should add all new flags here.

Reply:
It helps with the test, but I guess that's not a good excuse to put here :)
On the other hand, I don't think there's a real risk / down side to it either

Reply:
I'll merge now, but if you'd like me to remove this please lmk